core/internal/testutils/configtest/v2/general_config.go

```diff
@@ -59,6 +59,7 @@ func overrides(c *chainlink.Config, s *chainlink.Secrets) {
 
 	c.WebServer.SessionTimeout = models.MustNewDuration(2 * time.Minute)
 	c.WebServer.BridgeResponseURL = models.MustParseURL("http://localhost:6688")
+	c.WebServer.BridgeCacheTTL = models.MustNewDuration(0)
```

Comment:
Can this be omitted? It should get the default already.... :thinking: 
```suggestion
```

Reply:
Done